Obsidian.SourceGenerators/Packets/Attributes/AttributeFactory.cs

```diff
@@ -31,7 +31,7 @@ public static AttributeBehaviorBase[] ParseValidAttributesSorted(IEnumerable<Att
         }
 
         result.Sort((a, b) => a.Flag.CompareTo(b.Flag));
-        return result.ToArray();
+        return [.. result];
```

Comment:
Does this produse the same results as `.ToArray();`?

Reply:
Not at the moment. There is an open issue for it in Roslyn right now. What it actually generates is sub-optimal.

Reply:
oops yikes

Reply:
Please see this issue https://github.com/dotnet/roslyn/issues/70656

Reply:
So it does produce an array representation of the list, but it does it in a less optimal way compared to `.ToArray()`? If so I'd recommend changing those back to the old .ToArray calls (which was not just in this class, but in others as well. I'll add comments where this change was made to save you some time :)

Reply:
I believe there is a way to ignore warnings project-wide but I don't know off the top of my head